app/assets/javascripts/Components/Modals/submission_url_submit_modal.jsx

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import Modal from "react-modal";
+
+class SubmitUrlUploadModal extends React.Component {
+  state = {
+    newUrl: "",
+    newUrlAlias: "",
+  };
+
+  componentDidMount() {
+    Modal.setAppElement("body");
+  }
+
+  onSubmit = event => {
+    event.preventDefault();
+    this.props.onSubmit("https://youtu.be/dQw4w9WgXcQ", "");
```

Comment:
Make this actually send what is in the state. Even though we are not handling this in the backend just yet you can still make the front end work as intended.